pages/about.html

```diff
@@ -4,9 +4,79 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Yun Liu Intro</title>
+  <link rel="stylesheet" href="/style/style.css">
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <div id="logo"><a href="/pages/index.html" rel="about" class="logo">Home</a></div>
```

Comment:
Careful with your file paths! The `/pages/index.html` path does not actually work because it is an incorrect absolute path. Starting a path with a slash (/) will start the file search at the root directory of your entire file system (not your project).
To correct this, we can use a relative path instead. This will provide directions to the target file based on the current file's location.
In code this looks like: `./pages/index.html`. The dot at the beginning signals to start searching in the current directory.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This applies to links to the CSS files and for images as well.